scipy/linalg/matfuncs.py

```diff
@@ -668,3 +672,191 @@ def rounded_sign(x):
         return S0
     else:
         return S0, errest
+
+
+def is_matrix_symmetric(A, rtol=1e-05, atol=1e-08):
+    """
+    Returns True if a matrix is symmetric
+    Parameters
+    ----------
+    A : (N, N) array_like
+        Matrix to evaluate.
+    rtol : float
+        Relative tolerance.
+    atol : float
+        Absolute tolerance.
+    
+    Returns
+    -------
+    X : boolean
+        True if a matrix A is symmetric.
+
+    Examples
+    --------
+    from scipy.linalg import is_matrix_symmetric
```

Comment:
every example code snippet needs to have `>>>` at the start of the line and then needs the results printed in the console. See other examples in the docs.

Reply:
OK, done